spring-cloud-function-web/src/test/java/org/springframework/cloud/function/web/RestApplicationTests.java

```diff
@@ -343,12 +358,12 @@ private String sse(String... values) {
 		@Bean({ "uppercase", "transform", "post/more" })
 		public Function<Flux<String>, Flux<String>> uppercase() {
 			return flux -> flux.log()
-					.map(value -> "[" + value.trim().toUpperCase() + "]");
+					.map(value -> "(" + value.trim().toUpperCase() + ")");
```

Comment:
JUnit prints test failures with square brackets inserted. It was too confusing, when JSON arrays might be coming back.